Sources/Core/AWSClientRuntime/Protocols/Ec2Query/EC2QueryError.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import protocol ClientRuntime.BaseError
+import enum ClientRuntime.BaseErrorDecodeError
+import class ClientRuntime.HttpResponse
+import class SmithyXML.Reader
+
+public struct EC2QueryError: BaseError {
```

Comment:
The "base error" for use with EC2Query. Provides values for the common error fields.